app/scripts/templates/sign_in.mustache

```diff
@@ -84,8 +84,8 @@
       </form>
 
       <div class="links">
-          <a href="/reset_password" class="left reset-password">{{#t}}Forgot password?{{/t}}</a>
-          <a href="/signup" class="right sign-up">{{#t}}Create an account{{/t}}</a>
+          <a href="/reset_password" class="left reset-password" id="forgot-password">{{#t}}Forgot password?{{/t}}</a>
```

Comment:
What are your thoughts on the id names? Would we ever emit on non-link elements? I think that something like `sign_in.forgot-password-link` would give a very clear indication of what the user did over `sign_in.forgot-password` when looking at it from metrics side. One common thing I noticed with datadog is that we still have a hard time mapping what the metric value is to the event that caused it.